.ci/macos-steps.yaml

```diff
@@ -60,9 +60,7 @@ steps:
   displayName: 'Build'
 
 # Run tests via ctest.
-# The OMP_NUM_THREADS is to prevent thrashing on older versions of OpenBLAS
-# (0.3.26 and older) where OpenMP and pthreads aren't playing together well.
-- script: cd build && OMP_NUM_THREADS=1 CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
+- script: cd build && CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
```

Comment:
I think we need to set it here.

Reply:
Yep, I tried it to see if it made things faster but it didn't make any difference for test time... then I realized Homebrew distributes 0.3.28 so there is no issue.